packages/dds/task-manager/src/taskManager.ts

```diff
@@ -312,6 +314,34 @@ export class TaskManager extends SharedObject<ITaskManagerEvents> implements ITa
         return lockAcquireP;
     }
 
+    public subscribeToTask(taskId: string) {
+        const disconnectHandler = () => {
```

Comment:
I'm wondering if there's an opportunity to just call into volunteerForTask here (and repeat the call after disconnect/reconnect)?  The fewer places in the code that we submit ops from, the simpler it is to follow the logic I think.

Reply:
I think this may actually add some complexity because of the differences in how they handle disconnection